kafka-rest/src/test/java/io/confluent/kafkarest/v2/KafkaConsumerManagerTest.java

```diff
@@ -520,38 +544,48 @@ public void onCompletion(List<ConsumerRecord<ByteString, ByteString>> records, E
         return bootstrapConsumer(consumer, true);
     }
 
-    /**
-     * Subscribes a consumer to a topic and schedules a poll task
-     */
-    private List<ConsumerRecord<ByteString, ByteString>> bootstrapConsumer(final MockConsumer<byte[], byte[]> consumer, boolean toExpectCreate) {
-        List<ConsumerRecord<ByteString, ByteString>> referenceRecords =
-            Arrays.asList(
-                ConsumerRecord.create(topicName, ByteString.copyFromUtf8("k1"), ByteString.copyFromUtf8("v1"), 0, 0),
-                ConsumerRecord.create(topicName, ByteString.copyFromUtf8("k2"), ByteString.copyFromUtf8("v2"), 0, 1),
-                ConsumerRecord.create(topicName, ByteString.copyFromUtf8("k3"), ByteString.copyFromUtf8("v3"), 0, 2));
-
-        if (toExpectCreate)
-            expectCreate(consumer);
-        consumer.schedulePollTask(new Runnable() {
-            @Override
-            public void run() {
-                consumer.addRecord(new org.apache.kafka.clients.consumer.ConsumerRecord<>(topicName, 0, 0, "k1".getBytes(), "v1".getBytes()));
-                consumer.addRecord(new org.apache.kafka.clients.consumer.ConsumerRecord<>(topicName, 0, 1, "k2".getBytes(), "v2".getBytes()));
-                consumer.addRecord(new org.apache.kafka.clients.consumer.ConsumerRecord<>(topicName, 0, 2, "k3".getBytes(), "v3".getBytes()));
-            }
-        });
-
-        String cid = consumerManager.createConsumer(
-                consumer.groupName, ConsumerInstanceConfig.create(EmbeddedFormat.BINARY));
-
-        consumer.cid(cid);
-        consumerManager.subscribe(consumer.groupName, cid, new ConsumerSubscriptionRecord(Collections.singletonList(topicName), null));
-        consumer.rebalance(Collections.singletonList(new TopicPartition(topicName, 0)));
-        consumer.updateBeginningOffsets(singletonMap(new TopicPartition(topicName, 0), 0L));
-
-        return referenceRecords;
+  /**
+   * Subscribes a consumer to a topic and schedules a poll task
```

Comment:
nit: This no longer calls `schedulePollTask`, can you either update the Javadoc, or add an inline comment when adding the records how is this equivalent?

Reply:
Removed comment, as it is redundant.